table_pg.sql

```diff
@@ -222,4 +222,15 @@ DO $$ BEGIN
   END IF;
 END $$
 
+;
+
+DO $$ BEGIN
+  IF NOT EXISTS(select * from pg_tables where schemaname = 'public' and tablename = 'migration') THEN
+    CREATE TABLE migration (
+      key VARCHAR(100) NOT NULL,
+      PRIMARY KEY (key)
```

Comment:
Would be nice to add date

Reply:
What reason

Reply:
I also think about renaming of table 'migration' to 'property' with columns 'key string not null' 'value string not null'

Reply:
to be precise this table is migration__history_ so being able to see in which order updates are executed would be nice

Reply:
"I'm also thinking about renaming..."
why?

Reply:
storage of properties like as feature flags or for other purposes,
maybe already have plans for this...

Reply:
done